src/components/PlayerInventory.vue

```diff
@@ -8,15 +8,15 @@
 
 <script>
 export default {
-  data() {
-    return {};
-  },
   computed: {
     playerPos() {
       const pos = this.$store.getters.playerPos;
-      return pos ? pos : { x: 0, y: 0 };
+      return pos || { x: 0, y: 0 };
```

Comment:
Why not include this logic in the store getter instead?

Reply:
It should live there. It won't live here for much longer anyway as it's only there while I was debugging some other stuff. The reason for the change was just to fix a linting error.